src/components/Crypto/styles.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+
+export const CryptoContainer = styled.View`
+  alignself: center;
+  height: 100px;
+  width: 85%;
+  marginbottom: 10px;
+  borderbottomcolor: #c6c8d9;
```

Comment:
the colors should be stored in a theme file:

src/utils/theme.ts 

```
const theme = {
   colors: {
     white: "#ffffff",
     black: "#000000",
     red: .....
   }
}
export default theme;
```

and 

import theme from '......theme'

border-color: theme.colors.red